docs/security/05-05-production-profile.md

```diff
@@ -1,65 +1,53 @@
+### Oathkeeper settings for the production profile
 
-**Oathkeeper settings:**
+The production profile requires the following parameters in order to operate:
 
-| Parameter |  Description | Example value |
+| Parameter |  Description | Required value |
 |-------|-------|:--------:|
 | **oathkeeper.deployment.resources.limits.cpu** | Defines limits for CPU resources. | `800m` |
 | **oathkeeper.deployment.resources.requests.cpu** | Defines requests for CPU resources. | `200m` |
 | **hpa.oathkeeper.minReplicas** |  Defines the initial number of created Oathkeeper instances. | `3` |
-| **hpa.oathkeeper.maxReplicas** | Defines the maximum number of created Oathkeeper instances. | `10` |
 
-### Persistence modes
+### Persistence modes for the production profile
 
-The production profile for the OAuth2 server enables the use of a database, which can be one of the following options:
-  - A [preconfigured PostgreSQL](https://github.com/helm/charts/tree/master/stable/postgresql) database, which is installed together with the Hydra server.
+The production profile for the OAuth2 server enables the use of a custom database, which can be one of the following options:
   - A user-maintained database to which credentials are supplied.
   - A [GCP Cloud SQL](https://cloud.google.com/sql) instance to which credentials are supplied. In this case, an extra [gcloud-proxy](https://cloud.google.com/sql/docs/mysql/sql-proxy) deployment is created allowing secured connections.
 
-#### Internal PostgreSQL setup
-The database is created in the cluster as a StatefulSet and uses a PersistentVolume that is provider-specific. This means that the PersistentVolume used by the database uses the default StorageClass of the cluster's host provider.
-
-In order to use the database, the following configuration parameters have to be supplied:
-
-**General settings:**
-
-| Parameter | Description | Required value |
-|----------|-----------| :---: |
-| **global.ory.hydra.persistence.enabled** | Defines whether Hydra should use the `database` mode of operation. If `false`, it uses the `in-memory` mode. | `true` |
-| **global.ory.hydra.persistence.postgresql.enabled** | Defines whether Hydra should initiate the deployment of an in-cluster database. Set to `false` to use a self-provided database. If set to `true`, Hydra always uses an in-cluster database and ignores the custom database details. | `true` |
-| **hydra.hydra.autoMigrate** | Enables Hydra auto-migration feature, which prepares the database | `true` | 
-
-You can find an example of the required [configmap here](./assets/003-ory-db-postgres.yaml)
-
-#### Custom database
+#### User-maintained database
 
 Alternatively, you can use a compatible, custom database to store the registered client data. To use a database, you must create a Kubernetes Secret with the database password as an override for the Hydra OAuth2 server. The details of the database are passed using these parameters of the production profile override:
```

Comment:
Shouldn't we change `custom database` into `user-maintained database` here as well?